src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -3,22 +3,129 @@ title: 'Alerts'
 
-An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
\ No newline at end of file
+## Actions
+
+Actions are located in **Project Settings > Alerts > Rules > "New Alert Rules" or "Edit Rule"**.
+
+[{% asset notifications/new_alert_rule.png alt="Buttons to create a New Alert Rule or to Edit an existing rule." %}]({% asset notifications/new_alert_rule.png @path %})
+
+An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
+
+### Action Types
+
+Currently, there are three types of actions you can take:
+
+- Send a notification (for all legacy integrations)
+    - Per-Project Integrations are considered to be legacy integrations
+- Send a notification via {service}
+    - service is one of mail and any configured per-project integration
+- Send a notification to the {workspace} Slack workspace to {channel} and show tags {tags} in notification
+    - to clarify this relates to the global Slack integration, *not* the per-project legacy integration
+
+[{% asset notifications/alert_actions.png alt="Dropdown of notification actions." %}]({% asset notifications/alert_actions.png @path %})
+
+An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
+
+### Frequency
+
+- Perform these actions at most once every {interval} for an issue
+    - interval: 5 minutes, 10 minutes, 30 minutes, 60 minutes, 3 hours, 12 hours, 24 hours, one week, or 30 days
+    - Frequency configuration can get complicated. For example, if an issue changes state from resolved to unresolved, and then this issue is resolved, unresolved, and resolved again within 5 minutes, but your frequency threshold is 30 minutes, you'll only get one notification
+
+[{% asset notifications/alert_frequency.png alt="Dropdown of minute and hour options for frequency of notifications." %}]({% asset notifications/alert_frequency.png @path %})
+
+## Environment
+
+- All Environments
+- any one of your defined (and not hidden) environments
+
+[{% asset notifications/alert_environment.png alt="Dropdown of environments. Currently, only All Environments is an option." %}]({% asset notifications/alert_environment.png @path %})
+
+## Digests
+
+Sentry will automatically digest alerts sent by some services to avoid flooding your inbox with individual issue notifications. To control how frequently notifications are delivered, use the sliders.
+
+- Minimum delivery interval
+- Maximum delivery interval
+
+[{% asset notifications/alert_digest.png alt="Sliding time scale for intervals of notifications." %}]({% asset notifications/alert_digest.png @path %})
+
+## Notification Management
+
+You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page.
```

Comment:
here too - since the page is specifically talking about project alerts we could directly link to here https://sentry.io/settings/account/notifications/alerts/